assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -98,15 +98,27 @@ function DashboardSummaryWidget( { Widget, WidgetReportZero, WidgetReportError }
 	} );
 
 	if ( loading ) {
-		return <PreviewBlock width="100%" height="276px" />;
+		return (
+			<Widget>
+				<PreviewBlock width="100%" height="276px" />
+			</Widget>
+		);
 	}
 
 	if ( error ) {
-		return <WidgetReportError moduleSlug="adsense" error={ error } />;
+		return (
+			<Widget noPadding>
```

Comment:
The `noPadding` attribute isn't needed here:
```suggestion
			<Widget>
```